### DISCUSSION TITLE
Is it possible to change Ports?

### DISCUSSION THREAD

**Discussion by seanbts** [dec0dOS/zero-ui#55] [Q&A]
Hi @dec0dOS , is it possible to use different ports to 80 / 443 which are currently in use for other servers and cant open ports 80 and 443 to zero-ui server?

Cheers

**Comment by dec0dOS** [comment]
Yes, just change the port number in docker-compose file.
Please note that automatic ssl in caddy will only work with 80/443 pair.
If you want to change the ports and setup HTTPS with lets encrypt, you'll have to setup the reverse proxy yourself (nginx is the best option).
That is out of scope of ZeroUI project, so there will be no support.
